sklearn/gaussian_process/tests/test_gpc.py

```diff
@@ -260,3 +265,20 @@ def test_warning_bounds():
         "Increasing the bound and calling "
         "fit again may find a better value."
     )
+
+
+@pytest.mark.parametrize(
+    "params, TypeError, err_msg",
```

Comment:
I see this is a copy of:

https://github.com/scikit-learn/scikit-learn/blob/1d1aadd0711b87d2a11c80aad15df6f8cf156712/sklearn/gaussian_process/tests/test_gpr.py#L632

but I prefer to use something like this:

```suggestion
    "params, error_type, err_msg",
```

since `TypeError` is a already a name for a Python exception.

Reply:
do I need to edit the `test_gpr.py` too?